src/test/compile-fail/issue-26194.rs

```diff
@@ -12,7 +12,7 @@ struct S(String);
 
 impl S {
     fn f(self: *mut S) -> String { self.0 }
-    //~^ ERROR mismatched method receiver
```

Comment:
similarly here, I'm wondering if the information about "method receivers" is showing up, but in a different place now?